File: code-samples/python/feeds@products@status.py
import requests

url = "https://vendor-api-staging.jumia.com/feeds/products/status"

payload = {"products": [
        {
            "id": "string",
            "sellerSku": "string",
            "createdAt": "string",
            "businessClients": [
                {
                    "businessClientCode": "string",
                    "status": "string"
                }
            ]
        }
    ]}
headers = {"Authorization": "Bearer "}

response = requests.request("POST", url, json=payload, headers=headers)

print(response.text)

File: code-samples/java/feeds@products@status.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://vendor-api-staging.jumia.com/feeds/products/status"))
    .header("Authorization", "Bearer ")
    .method("POST", HttpRequest.BodyPublishers.ofString("{\n  \"products\": [\n    {\n      \"id\": \"string\",\n      \"sellerSku\": \"string\",\n      \"createdAt\": \"string\",\n      \"businessClients\": [\n        {\n          \"businessClientCode\": \"string\",\n          \"status\": \"string\"\n        }\n      ]\n    }\n  ]\n}"))
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());